charts/rawls/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: rawls
-version: 0.20.0
+version: 0.21.0
```

Comment:
Why did you bump this version? Doesn't chart releaser do this for us?

Reply:
I bumped it here to give me a fresh version number, I was having trouble getting Argo to respect the versions I was giving it through helmfile. I know that the chart releaser will bump it again